core/src/components/buttons/buttons.ios.scss

```diff
@@ -81,7 +81,7 @@
   @include margin(0);
   @include margin-horizontal(null, .3em);
 
-  font-size: 24px;
+  font-size: 1.4em;
```

Comment:
This value is calculated off of `17px` which is the default `font-size` here, not `16px`